packages/web-app-files/src/components/Shares/SharedWithMeSection.vue

```diff
@@ -48,7 +48,7 @@
             v-if="getShowDeclineButton(resource)"
             size="small"
             class="file-row-share-decline oc-ml-s"
-            @click.stop="triggerAction('decline-share', { resources: [resource] })"
+            @click.stop="triggerAction('decline-share', { space: null, resources: [resource] })"
```

Comment:
Why is space`null` here? Shouldn't it be `createShareSpace(resource)` like down below?

Reply:
Well it's unused in the action handler, so I didn't bother to get a space from somewhere

Reply:
As discussed: Let's leave it (for now). The action does not need it and we don't risk any performance impact with hundreds of shares.

Reply:
Quickly discussed in chat: We are going to stick with null for now as it's unneeded and it might become a performance problem when having A LOT of spaces. It could be optimized when needed but right now we don't think it makes sense.